Hydrogent/CMakeLists.txt

```diff
@@ -2,6 +2,9 @@ cmake_minimum_required (VERSION 3.13)
 
 project(Diligent-Hydrogent CXX)
 
+set(pxr_DIR ${DILIGENT_USD_PATH})
+find_package(pxr)
```

Comment:
Where is this module defined?
This does not seem to be a [standard CMake module](https://cmake.org/cmake/help/latest/manual/cmake-modules.7.html)

Reply:
It is generated when OpenUSD is built and deployed, so something like `${pxr_PATH}/pxrConfig.cmake` and it'll point to its dependencies, in my case I build with boost::python and needed them to link to USDViewer.

Reply:
I am wondering how this will work on other platforms.
I think it should be guarded by `if(PLATFORM_LINUX)`

Reply:
On Windows we build with these commands:
https://github.com/DiligentGraphics/DiligentFX/tree/master/Hydrogent#build-instructions

Reply:
Ok if it is working on windows I don't want to break things, I've wrapped it with `if(PLATFORM_LINUX)` for now and put a comment on there what it does, but I'll try later on windows when I get access to a machine.